### ISSUE TITLE
Layout Grid: How to use with the Query block

### ISSUE THREAD

**Issue by nebulousGirl** [Automattic/block-experiments#271]
I wish to use the Layout Grid plugin with the Query block to create an archive page. Is there a way to make it work as is?

**Comment by jasmussen** [comment]
Indeed, that appears to not be possible at the moment, I think that needs a change [upstream](https://github.com/WordPress/gutenberg/) as you can't use columns either.
